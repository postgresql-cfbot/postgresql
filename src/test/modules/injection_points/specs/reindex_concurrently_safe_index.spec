setup
{
	CREATE EXTENSION injection_points;
	CREATE SCHEMA test;
	CREATE TABLE test.tbl(i int primary key, updated_at timestamp);
	CREATE UNIQUE INDEX tbl_pkey_not_safe ON test.tbl(ABS(i)) WHERE MOD(i, 2) = 0;
}

teardown
{
	DROP SCHEMA test CASCADE;
	DROP EXTENSION injection_points;
}

session test
setup	{
	SELECT injection_points_set_local();
	SELECT injection_points_attach('ReindexRelationConcurrently_index_safe', 'error');
}
step reindex	{ REINDEX INDEX CONCURRENTLY test.tbl_pkey_not_safe; }

permutation
	reindex