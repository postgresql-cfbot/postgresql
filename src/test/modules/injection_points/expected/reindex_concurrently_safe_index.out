Parsed test spec with 1 sessions

starting permutation: reindex
injection_points_attach
-----------------------
                       
(1 row)

step reindex: REINDEX INDEX CONCURRENTLY test.tbl_pkey_not_safe;
